framework/wazuh/core/authentication.py

```diff
@@ -62,10 +18,16 @@ def get_keypair() -> tuple[str, str]:
     public_key : str
         Public key.
     """
-    config = CentralizedConfig.get_server_config()
+    config = CentralizedConfig.get_management_api_config()
```

Comment:
This function is used in both APIs, but given that the keys will be the same, we could use any of the two configurations